### PR TITLE
Update configure-unit-tests-by-using-a-dot-runsettings-file.md

### PR DESCRIPTION
**Pull request by engyebrahim**
related to : https://github.com/microsoft/testfx/issues/1574

### CODE REVIEW

**Comment by Court72** [comment]
@Mikejo5000 

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md [hunk 1]
```diff
@@ -322,7 +322,7 @@ These settings are specific to the test adapter that runs test methods that have
 |**DeploymentEnabled**|true|If you set the value to **false**, deployment items that you've specified in your test method aren't copied to the deployment directory.|
 |**CaptureTraceOutput**|true|You can write to the debug trace from your test method using <xref:System.Diagnostics.Trace.WriteLine%2A?displayProperty=nameWithType>.|
 |**EnableBaseClassTestMethodsFromOtherAssemblies**|true|A value indicating whether to enable discovery of test methods from base classes in a different assembly from the inheriting test class.|
-|**ClassCleanupLifecycle**|EndOfClass|If you want the class cleanup to occur at the end of assembly, set this to **EndOfAssembly**.|
+|**ClassCleanupLifecycle**|EndOfClass|If you want the class cleanup to occur at the end of assembly, set this to **EndOfAssembly**. (No longer supported starting from MSTest v4 as EndOfClass is the default and only ClassCleanup behavior)|
```
Comment (Mikejo5000) [inline docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md:325]: @engyebrahim Can we make "ClassCleanup" a link?

Comment (Mikejo5000) [inline docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md:325]:
```suggestion
|**ClassCleanupLifecycle**|EndOfClass|If you want the class cleanup to occur at the end of assembly, set this to **EndOfAssembly**. (No longer supported starting from MSTest v4 as EndOfClass is the default and only <xref:Microsoft.VisualStudio.TestTools.UnitTesting.ClassCleanupAttribute> behavior)|
```

**Comment by Evangelink** [comment]
@Mikejo5000 MSTest v4 is not yet available so maybe you want to keep this PR on-hold for now. We will ping you once v4 is released.

Note that if that's fine for you, I don't mind merging it now.

**Review by Mikejo5000 — APPROVED** [review]

**Comment by Mikejo5000** [comment]
#sign-off

**Comment by Jak-MS** [comment]
@engyebrahim @Mikejo5000 - same comment as in the other PR. you'll need to verify that the xref: link is working after this is published.

**Pull request merged**